src/pages/registration/SectionForm.tsx

```diff
@@ -48,6 +48,16 @@ type SectionFormParams = {
   onScreen: (slotAddress: string) => void;
 };
 
+export const LifeStageMap: Record<string, string> = {
+  ...Object.entries(LifeStage).reduce(
+    (acc, [key, value]) => {
+      acc[key] = value;
+      return acc;
+    },
+    {} as Record<string, string>
+  ),
+  'N/A': 'n/a'
```

Comment:
What do the two versions of N/A represent here?

Reply:
n/a for not applicable, I suppose, that's the ticket's request

Reply:
Yes, why are there two versions 'N/A' and 'n/a'? What do the two different versions represent?

Reply:
oh, as the uppercase is for label, and the lowercase is for key, just to match how the other life stage values are set.

Reply:
I checked it with the core and it looks like LifeStage is a required field. I am updating the branch, to send null to the core, but probably you need to do some work at your end.

Reply:
Yep there is a pull request in core to make LifeStage nullable. 👍